libsoftwarecontainer/src/softwarecontainer.cpp

```diff
@@ -81,16 +79,13 @@ void SoftwareContainer::setContainerName(const std::string &name)
     log_debug() << m_container->toString();
 }
 
-void SoftwareContainer::validateContainerID()
+ReturnCode SoftwareContainer::preload()
 {
-    if (m_containerID.size() == 0) {
-        setContainerIDPrefix("PLC-");
+    if (m_containerID.empty()) {
+        const std::string newID = "SC-PRELOADED-" + Generator::gen_ct_name();
+        setContainerID(newID);
     }
```

Comment:
Can we have all IDs (and thus also LXC names) use the same scheme? I.e. have the preloaded containers also use <prefix>-<next-available-id>?